src/emails/receiving/attachments/interfaces/get-attachment.interface.ts

```diff
@@ -0,0 +1,21 @@
+import type { ErrorResponse } from '../../../../interfaces';
+import type { Attachment } from './attachment';
+
+export interface GetAttachmentOptions {
+  emailId: string;
+  id: string;
+}
+
+export type GetAttachmentResponseSuccess = {
+  object: 'attachment';
```

Comment:
This need to be changed
```suggestion
  object: 'email';
```

Reply:
Why? Isn't it an `attachment`? Am I missing something?

Reply:
The entity is email, not attachment